src/components/molecules/ArtifactsList/ArtifactsList.tsx

```diff
@@ -55,7 +62,27 @@ const ArtifactsList: React.FC<ArtifactsListProps> = props => {
     });
   }, [artifacts, testExecutionId, isLoading]);
 
-  return <ArtifactsListContainer>{renderedArtifactsList}</ArtifactsListContainer>;
+  const handleDownloadAll = async () => {
+    try {
+      setIsDownloading(true);
+      await downloadArtifactArchive(`${testName}-${testExecutionId}`, testExecutionId);
```

Comment:
Maybe it's worth to add a date to this name?
```
      await downloadArtifactArchive(`${testName}-${testExecutionIsoDateOrTimestamp}-${testExecutionId}`, testExecutionId);
```

Reply:
Maybe, but this file name is still pretty big. What benefits of adding timestamps here?

Reply:
When the User will download artifacts from multiple executions and will sort the files, will get a list from the latest to the oldest

Reply:
Actually, I think that most often the execution ID is not random ID - it's `test-name-1234`. It's not always happening though, as:
* I've seen sometimes the prefix missing
* The execution name may be overridden

Reply:
Ok